docs/user/lfs.rst

```diff
@@ -35,6 +35,24 @@ JupyterLab sessions by default. As a result, you do need to be aware of dealing
 with data stored in LFS if you want to use it efficiently in your work with
 renku.
 
+Renku LFS configuration
+-----------------------
+Renku by default only stores files larger than 100kb in LFS, to prevent
```

Comment:
Renku by default stores all files larger than 100kb in LFS to prevent slowing down git (and thus `renku`) with large files.